content/applications/general/email_communication/email_servers.rst

```diff
@@ -99,100 +85,39 @@ message, Odoo will try to redistribute that same email to the other subscribers
-* **Original messages**: Several business objects have their own alias to create new records in
-  Odoo from incoming emails:
-
-  * Sales Channel (to create Leads or Opportunities in `Odoo CRM <https://www.odoo.com/page/
-    crm>`__),
-  * Support Channel (to create Tickets in `Odoo Helpdesk <https://www.odoo.com/page/helpdesk>`__),
-  * Projects (to create new Tasks in `Odoo Project <https://www.odoo.com/page
-    /project-management>`__),
-  * Job Positions (to create Applicants in `Odoo Recruitment <https://www.odoo.com/page
-    /recruitment>`__),
-  * etc.
-
-Depending on the mail server, there might be several methods to fetch emails. The easiest and most
-recommended method is to manage one email address per Odoo alias in the mail server.
-
-* Create the corresponding email addresses in the mail server (catchall@, bounce@, sales@, etc.).
-* Set the :guilabel:`Alias Domain` name in :menuselection:`Settings --> General Settings -->
-  Discuss`.
-* If the database's hosting type is Odoo on-premise, create an :guilabel:`Incoming Mail Server` in
-  Odoo for each alias. This can be done from the General Settings as well. Fill out the form
-  according to the email provider's settings. Leave the :guilabel:`Actions to Perform on Incoming
-  Mails` field blank. Once all the information has been filled out, click on :guilabel:`TEST &
-  CONFIRM`.
-
-  .. image:: email_servers/incoming-server.png
-     :align: center
-     :alt: Incoming mail server configuration on Odoo.
+Access the :guilabel:`System Parameters` by activating :ref:`developer mode <developer-mode>` and
+going to :menuselection:`Settings --> Technical --> Parameters --> System Parameters` menu.
 
-* If the database's hosting type is Odoo Online or Odoo.sh, redirecting incoming messages to Odoo's
-  domain name instead of the external email server is recommended. That way, incoming messages can
-  be received without delay. Odoo Online only fetches incoming messages of external servers once
-  every hour. Redirections for all email addresses should be set to Odoo's domain name in the email
-  server (e.g. *catchall\@mydomain.ext* to *catchall\@mycompany.odoo.com*).
+To force the email address from which emails are sent, a combination of the following keys needs to
+be set in the system parameters of the database:
 
-All the aliases are customizable in Odoo. Object aliases can be edited from their respective
-configuration view.
+- `mail.default.from`: accepts the local part or a complete email address as value
+- `mail.default.from_filter`: accepts a domain name or a full email address as value
 
-.. tip::
-   To edit catchall and bounce aliases, first activate the :ref:`developer mode <developer-mode>`.
-   Then, go to :menuselection:`Settings --> Technical --> Parameters --> System Parameters` to
-   customize the aliases (*mail.catchall.alias* & *mail.bounce.alias*).
+.. note::
+   The `mail.default.from_filter` works only for `odoo-bin` configurations, otherwise this
```

Comment:
The odoo-bin configuration is not defined in the document, is this concept clear enough for the reader? While waiting for the CLI part to be added, I would use a more informative denomination such as "the default Odoo mail server".

Reply:
@jqu-odoo Thanks for pointing this out. I made a reference to the CLI doc which talks about the odoo_bin. Is that okay in your opinion? Otherwise I can change -- it is my understanding that the system parameter: `mail.default.from_filter` also works for custom email servers as well as the default Odoo mail server. Can you clarify a bit / make a suggestion to what it should be changed to? I agree there is some complexity to the original statement. Thanks :)

Reply:
Hey @tiku-odoo I can confirm that mail.default.from_filter works only with the default Odoo mail server, not with custom mail servers. Indeed you can call it odoo-bin if there is a link to the related documentation ;-)

Reply:
Can you check the configuration you left on this db: https://www.purpletest.net ? The mail.default.from_filter system parameter is present. I've changed the doc to reflect what you have noted. Thanks for pointing this out!

Reply:
The config parameter is there but it does nothing because there is an enabled mail server. I removed the mail.default.from_filter from your database to show that the from is still encapsulated.